### PR TITLE
Hide clusters tab with CSS

### PR DESCRIPTION
**Pull request by ivanhercaz**
See [T120629](https://phabricator.wikimedia.org/T120629) in Phabricator.

As @chicocvenancio said in the Phabricator task mentioned above, the CSS snippet written by @framawiki is simpler and more specific than the suggested in #22.

### CODE REVIEW

**Comment by ivanhercaz** [comment]
I think the checks failed due to some issue related with Docker, but at this moment I don't know how I could fix it @chicocvenancio :confused:

File: images/singleuser/Dockerfile [hunk 1]
```diff
@@ -141,6 +141,11 @@ RUN pip install --no-cache-dir git+https://github.com/yuvipanda/nbpawspublic@023
 
 RUN pip install --no-cache-dir git+https://github.com/yuvipanda/ipynb-paws@60c94e3
 
+# use custom css to hide clusters tab
+COPY hide_clusters_tab.css /home/paws/.jupyter/custom/custom.css
```
Comment (chicocvenancio) [inline images/singleuser/Dockerfile]: COPY now allows you to have a ` --chown=tools.paws:tools.paws ` modifier and save a few layers.

**Comment by chicocvenancio** [comment]
> I think the checks failed due to some issue related with Docker, but at this moment I don't know how I could fix it @chicocvenancio 😕

Yeah, I fixed the image issues now. Once you address the separate chown we can merge and deploy this.

File: images/singleuser/Dockerfile (hunk 1 above)
Comment (ivanhercaz) [reply images/singleuser/Dockerfile]: I change it to use the ``chown`` modifier. Is the Dockerfile fines now?

**Pull request merged**